packages/interledger/src/accounts/service.ts

```diff
@@ -102,6 +105,7 @@ function toIlpAccount(accountRow: IlpAccountModel): IlpAccount {
 interface BalanceOptions {
   id: bigint
   flags: number
+  unit: number
```

Comment:
What does `unit` actually refers to? I understand it corresponds to a field in tigerbeetle... Is it a tigerbeetle-defined unique identifier of a scale/code tuple?

Reply:
https://github.com/coilhq/tigerbeetle/blob/main/docs/DESIGN.md#events

>unit:  2 bytes ( 16-bit) [optional, opaque unit of value, e.g. a currency code, or even something exotic like gold bars]

We're using it to represent an asset code + scale.
Tigerbeetle disallows transfers between accounts with different `unit`s

Reply:
Ok, but I mean where is the actual value for it generated? Or do we just hard-code a mapping somewhere, e.g. USD:9 :: 0x1234?

Reply:
It's the [`smallserial`](https://www.postgresql.org/docs/current/datatype-numeric.html#DATATYPE-SERIAL) `unit` column in the assets table:
https://github.com/interledger/rafiki/pull/117/commits/8a39d388028adbc87f680a9e14cb700147f5d801#diff-311f81ba7c657dbb0ad94d4b0c226dc9649478d55578f579ed72047aa581ffebR6

Reply:
Ahh, I see, so it autoincrements. :+1: